utils/modeler/modeler.go

```diff
@@ -70,13 +79,30 @@ func (m *Modeler) mapToModel(data map[string]string, context string, rv reflect.
 		} else {
 			// We're not nested!
 			var key string
+			prefix := m.prefix
+			if prefix != "" && !strings.HasSuffix(prefix, "/") {
+				prefix = fmt.Sprintf("%s/", prefix)
+			}
 			if context == "" {
-				key = fmt.Sprintf("%s/%s", m.prefix, tagValue)
+				key = fmt.Sprintf("%s%s", prefix, fieldTagValue)
 			} else {
-				key = fmt.Sprintf("%s/%s.%s", m.prefix, context, tagValue)
+				key = fmt.Sprintf("%s%s.%s", prefix, context, fieldTagValue)
 			}
 			stringVal, ok := data[key]
 			if ok {
+				constraintTagValue := rf.Tag.Get(m.constraintTag)
+				if constraintTagValue != "" {
+					constraint := regexp.MustCompile(constraintTagValue)
```

Comment:
Sorry I'm late to the party. This could panic, so maybe better to just call [`Compile`](https://godoc.org/regexp#Compile) and check for the error?